eureka2-server/src/main/java/com/netflix/eureka2/server/health/HealthStatusProviderRegistry.java

```diff
@@ -32,7 +34,8 @@ public void add(HealthStatusProvider<?> provider) {
         return providerSubject;
     }
 
-    @PostInjector
+//    @PostInjector
```

Comment:
With governator 1.7 we can @ Inject LifecycleManager and then add a listener for onStarted(). Changing this to avoid needing @ PostInject all together.